config/apply/ccsm/translate.js

```diff
@@ -323,6 +345,9 @@ export function translateResponse(customApiResponse, patientData) {
       codings.push(standardTestTypeCodes['Cervical Histology']);
     }
 
+    // Is findingType always provided?
```

Comment:
> // Is findingType always provided?

I do not know. It is provided in every example provided to us, but we would need to check with the pilot technical team to confirm. Keeping the `?` in this line is safe for now.